__tests__/test.test.js

```diff
@@ -307,4 +307,85 @@ describe('POST /api/articles/:article_id/comments', () => {
         expect(body.comment).not.toHaveProperty('unnecessaryProperty')
         })
     })
-})
\ No newline at end of file
+})
+
+describe('PATCH /api/articles/:article_id', () => {
+    test('Status code 200 returned', () => {
+        return request(app)
+        .patch('/api/articles/1')
+        .send({ inc_votes: 1 })
+        .expect(200)
+        })
+
+    test('Patch responds with an object that has the correct properties', () => {
+        return request(app)
+        .patch('/api/articles/1')
+        .send({ inc_votes: 5 })
+        .then(({ body }) => {
+        expect(body.article).toBeInstanceOf(Object)
+        expect(body.article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String)
+        })
+    })
```

Comment:
I would be more specific here - your test isn't proving that you've managed to patch **this** article. At the minimum I would add what article_id you expect to come back.